src/Features/CustomerEffortScoreTracks.php

```diff
@@ -35,6 +37,10 @@ public static function get_instance() {
 	 * Constructor. Sets up filters to hook into WooCommerce.
 	 */
 	public function __construct() {
+		if ( ! Loader::is_feature_enabled( 'customer-effort-score' ) ) {
```

Comment:
Suggestion: Put the the feature check in the implementing code. For example:
```php
if ( Loader::is_feature_enabled( 'customer-effort-score' ) ) {
  new CustomerEffortScoreTracks()
}
```

Putting it here means we can't use the class in a standalone way. This can make unit tests for the class brittle since they would be reliant on `Loader` being in a certain state.

Reply:
ah of course, thanks